cmds/less/main.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// less pages through files
+//
+// Synopsis:
+//     less [OPTIONS] FILES...
+//
+// Options:
+//     -profile FILE: Save profile in this file
+//     -tabstop NUMBER: Number of spaces per tab
+//
+// Keybindings:
+//     Control:
+//
+//     * q: Quit
+//
+//     Scrolling:
+//
+//     * j: Scroll down
+//     * k: Scroll up
+//     * g: Scroll to top
+//     * G: Scroll to bottom
+//     * Pgdn: Scroll down one screen full
+//     * Pgup: Scroll up one screen full
+//     * ^D: Scroll down one half screen full
+//     * ^U: Scroll up one half screen full
+//
+//     Searching:
+//
+//     * /: Enter search regex (re2 syntax). Press enter to search.
+//     * n: Jump down to next search result
+//     * N: Jump up to previous search result
+//
+// Author:
+//     Michael Pratt (github.com/prattmic)
```

Comment:
Perhaps note this is originally based on https://github.com/prattmic/lesser? Though my repo is abandoned, so this will effectively become the canonical version,